themes/gruvbox-dark-hard.json

```diff
@@ -988,6 +988,11 @@
     "editorError.foreground": "#cc241d",
     "editorWarning.foreground": "#d79921",
     "editorInfo.foreground": "#458588",
+    // EDITOR - INLAY HINTS
+    "editorInlayHint.background": "#3c383680",
+    "editorInlayHint.foreground": "#928374",
+    "editorInlayHint.typeForeground": "#d79921",
+    "editorInlayHint.paramForeground": "#458588",
```

Comment:
This should be `parameterForeground`:
https://github.com/microsoft/vscode/blob/2f582604fbdcedf1deff493ff7afa996d0d68c0f/src/vs/platform/theme/common/colorRegistry.ts#L391-L396

Reply:
Fixed